packages/xo-web/src/xo-app/new/network/index.js

```diff
@@ -0,0 +1,257 @@
+import _, { messages } from 'intl'
+import ActionButton from 'action-button'
+import decorate from 'apply-decorators'
+import Page from '../../page'
+import PropTypes from 'prop-types'
+import styles from './index.css'
+import React, { Component } from 'react'
+import Wizard, { Section } from 'wizard'
+import { map } from 'lodash'
+import { Toggle } from 'form'
+import { connectStore } from 'utils'
+import { injectIntl } from 'react-intl'
+import { linkState } from 'reaclette-utils'
+import { Container, Row, Col } from 'grid'
+import { injectState, provideState } from 'reaclette'
+import { SelectPif, SelectPool } from 'select-objects'
+import { createGetObject, getIsPoolAdmin } from 'selectors'
+import { createBondedNetwork, createNetwork, getBondModes } from 'xo'
+
+const getInitialState = () => ({
+  bonded: false,
+  bondMode: undefined,
+  description: '',
+  mtu: '',
+  name: '',
+  pif: undefined,
+  pifs: undefined,
+  vlan: '',
+})
+
+const NewNetwork = decorate([
+  connectStore(() => ({
+    isPoolAdmin: getIsPoolAdmin,
+    pool: createGetObject((_, props) => props.location.query.pool),
+  })),
+  injectIntl,
+  provideState({
+    initialState: () => ({ ...getInitialState(), bondModes: undefined }),
+    effects: {
+      linkState,
+      initialize: async () => {
+        const bondModes = await getBondModes()
+        return { bondModes }
+      },
+      onChangeBondedNetwork: () => ({ bonded, bondModes }) => ({
+        ...getInitialState(),
+        bonded: !bonded,
+        bondMode: !bonded ? bondModes[0] : undefined,
```

Comment:
`bondModes` may be `undefined`.
Why do you need to auto-select a bond mode?

Reply:
The `select` in UI has a first option selected but `bondMode` (`select`'s value) is undefined.

Reply:
Then we should probably either use `form/Select` or add an empty option `<option disabled>Select a bond mode</option>` in the options list and set `bondMode: ''` in the empty state. Otherwise, we'll have issues with the bond modes list being provided asynchronously.